Yoast/javascript
# Add Indonesian pseudo-plurals

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds entries for Indonesian pseudo-plurals, both for non-stem-changing words (e.g., aba-aba, meminta-minta -> minta-minta) and for those that do change (e.g., memutar-mutar -> putar-putar), and related specs. Also the goldStandard list was updated to accomodate the changes.
* [Yoast SEO Premium] Adds recognition for Indonesian words that look like plurals but are not (for example: aba-aba, putar-putar).

## Relevant technical choices:

* The stem-changing words are not listed one after the other. For more details see the accompanying [Premium Configuration PR](https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/194).

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all test pass.
* Try adding the following words to specs and see i you get the correct stem:
   - Stems undergoing modification:
        - mengedek-ngedek -> kedek-kedek;
        - memilih-milih -> pilih-pilih;
        - menyabu-nyabu -> sabu-sabu;
        - menakut-nakuti ->  takut-takut.
  
  - Stem not undergoing modification:
       - aba-aba -> aba-aba;
       - barik-barik -> barik-barik;
       - cawi-cawi -> cawi-cawi;
       - gail-gail -> gail-gail.

 
   - Rule for stem modification in case you want to try out more words from [this list](https://docs.google.com/spreadsheets/d/1DgUpjotwkNzOppb_iNwyi15hPDPY6dwHSLG_RH1Vmz4/edit#gid=0): k-> ng, p->m, s->ny, t->n plus adding "me" before the first half of the word (even though the words can also take other affixes: the safes option is asking @FAMarfuaty ).

* In the content-analysis app: add forms from the previous step to the keyphrase and to the title/text/meta description and see whether they are recognised.

* QA Testing instructions

Make sure all of the following forms are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description:
   - Two forms for one word:
        - mengedek-ngedek -> kedek-kedek;
        - kedek-kedek -> kedek-kedek;
        - memilih-milih -> pilih-pilih;
        - pilih-pilih -> pilih-pilih;
        - menyabu-nyabu -> sabu-sabu;
        - sabu-sabu -> sabu-sabu;
        - menakut-nakuti ->  takut-takut;
        - takut-takut -> takut-takut;

  - One form for one word:
       - aba-aba -> aba-aba;
       - barik-barik -> barik-barik;
       - cawi-cawi -> cawi-cawi;
       - gail-gail -> gail-gail.



## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-432
